api/pb/android/threads/src/main/java/io/textile/threads/Session.java

```diff
@@ -0,0 +1,6 @@
+package io.textile.threads;
+
+class Session {
```

Comment:
what's the `id` here? presumably it's not the session id since that is probably stored in `session_id`?

Reply:
Good question, but I also don't see that this class is used anywhere. Could remove if that's true.

Reply:
Oh, I see it's used in `TextileConfig.refreshSession`.

Reply:
It's only an internal class